1.10/administering-clusters/deploying-a-local-dcos-universe.md

```diff
@@ -207,7 +207,24 @@ You can install and run DC/OS services on a datacenter without internet access w
     sudo systemctl restart docker
     ```
 
-1.  Close the SSH session by typing `exit` or open a new terminal prompt. Repeat steps 28 and 29 on each agent node.
+1. Configure the Apache Mesos fetcher to trust the downloaded Docker certificate.
+
+   1. Copy the certificate:
+   ```
+   sudo cp /etc/docker/certs.d/master.mesos:5000/ca.crt /var/lib/dcos/pki/tls/certs/ca.crt
```

Comment:
The `/var/lib/dcos/pki/tls/certs/` directory is allowed to contain many certificate files. Each file must have its own unique filename in this directory, and that filename does not actually matter for anything. There just must not be any clash. `ca.crt` is a little too generic, I find. Can we call this `/var/lib/dcos/pki/tls/certs/docker-registry-ca.crt`?